.github/workflows/publish.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
     - name: Update changelog
       run: |
-        (head -8 CHANGELOG.md && echo "## $VERSION" && date "+_%e %B %Y_" && echo "" && (echo "$CHANGELOG" | sed -E 's_\(#([0-9]+)\)_([#\1](https://github.com/contentauth/c2pa-rs/pull/\1)\)_') && echo "" && tail -n +9 CHANGELOG.md) > CHANGELOG.new.md
```

Comment:
The `echo ""` step was intentional to have a consistent whitespace between releases in the changelog.

Reply:
not sure how that got changed! will add it back